The deletingAnnouncement object doesn't have a title field

olimorris opened the pull request:
In it's current form, when you click to delete an announcement, the modal pops up with `Deleting Announcement ()`. This is due to the incorrect use of:

```
Delete Announcement (@{{ deletingAnnouncement.title }})
```

The object does not have a title property.

aaranmcguire commented:
Tested, Working.

cwt137 commented:
Hmm. I would like to see this done in the regular framework instead of Spark. Also, a login failure should trigger an event.

aaranmcguire commented:
Looking on what the best way to trigger a login failure.

aaranmcguire commented:
Just looked in the Laravel Framework. It seems this already exists, it's just not advertised. and its not using classes at the fire 'name'.

`auth.attempt`
`auth.login`
`auth.logout`

Commit:
The deletingAnnouncement object doesn't have a title field

In it's current form, when you click to delete an announcement, the modal pops up with `Deleting Announcement ()`. This is due to the incorrect use of:

```
Delete Announcement (@{{ deletingAnnouncement.title }})
```

The object does not have a title property.
